jac-uk/admin
# #2008 Exercise Approval - Feedback Chain

## What's included?
For rejected exercises allow a requester to add a note in response to a rejection message.
Display the rejection response to the approver.

Closes #2008 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?

- Create an exercise
- Submit exercise for approval
- A modal should popup with NO link saying 'Add a note to the approver?'
- In the modal click the Submit For Approval button
- Login as an approver (ie selection mgr)
- Go to the exercise
- Reject the exercise and add a rejection message (message 1)
- Login as the requester
- Go to the exercise
- See message 1 telling you it has been rejected
- Scroll down and click the Submit For Approval button
- See the modal with a link saying 'Add a note to the approver?'
- Click the link saying 'Add a note to the approver?' and add a message (message 2) for the approver then click the Submit For Approval button
- Login as an approver (ie selection mgr)
- Go to the exercise and you should see two message bubbles containing message 1 and message 2 and the option to Reject or Approve
- Approve the exercise

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
